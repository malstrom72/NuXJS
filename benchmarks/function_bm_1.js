function a(x) {
    function b(y) {
        function c(z) {
            return y + z;
        }
        return x + y + c(x * y);
    }
    return x + b(x * x);
}

for (var i = 0; i < 1000000; ++i) {
    var v = a(i);
}
