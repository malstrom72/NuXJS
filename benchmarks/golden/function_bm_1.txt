999999000001000000
